### PR TITLE
gh-564: removing local janusgraph image

### PR DESCRIPTION
**Pull request by idcrosby**
Addresses: https://github.com/creativesoftwarefdn/weaviate/issues/564

### CODE REVIEW

**Comment by bobvanluijt** [comment]
@idcrosby Travis fails, do you want to fix this first or would you prefer me to merge?

File: docker-compose.yml [hunk 1]
```diff
@@ -53,13 +53,7 @@ services:
...
     restart: on-failure:0
   janus:
-    image: janusgraph:0.2.0-hadoop2
-    build:
-      dockerfile: Dockerfile.janus
-      args:
-        version: 0.2.0
-        hadoop: hadoop2
-      context: ./tools/dev/janusgraph-docker/
+    image: creativesoftwarefdn/janusgraph-docker
```
Comment (moretea) [inline docker-compose.yml]: Could you update this to refer to a tagged version?

**Comment by moretea** [comment]
Also you need to make the image publicly available?

It fails with the following error message:

```
Pulling janus (creativesoftwarefdn/janusgraph-docker:latest)...
pull access denied for creativesoftwarefdn/janusgraph-docker, repository does not exist or may require 'docker login'
```
https://travis-ci.org/creativesoftwarefdn/weaviate/jobs/455595326#L729

**Comment by moretea** [comment]
@bobvanluijt so please don't merge this yet, as it will break the development branch.

**Comment by bobvanluijt** [comment]
> @bobvanluijt so please don't merge this yet, as it will break the development branch.

Check, I'll wait for the go.

File: docker-compose.yml (hunk 1 above)
Comment (idcrosby) [reply docker-compose.yml]: I guess the easiest would be to tag it with the version of janusgraph?

Comment (idcrosby) [reply docker-compose.yml]: see: https://github.com/creativesoftwarefdn/janusgraph-docker/issues/5

**Comment by idcrosby** [comment]
Docker Hub automated builds have been enabled. https://hub.docker.com/r/creativesoftwarefdn/janusgraph-docker/tags/

File: docker-compose.yml (hunk 1 above)
Comment (idcrosby) [reply docker-compose.yml]: fixed

**Comment by idcrosby** [comment]
This should be good to merge in now. cc @moretea  @bobvanluijt

**Pull request merged**